sdks/python/apache_beam/transforms/external.py

```diff
@@ -750,7 +751,7 @@ def expand(self, pvalueish):
     with ExternalTransform.service(expansion_service) as service:
       response = service.Expand(request)
       if response.error:
-        raise RuntimeError(response.error)
+        raise RuntimeError(_sanitize_java_traceback(response.error))
```

Comment:
For transform service, we combine the errors from all expansion service container and return them together. Usually one of them is useful while other is a trivial error (for example, URN not available). Seems like this might result in useful Python errors being deleted in that case ?

https://github.com/apache/beam/blob/54db453b032bc3e4827fd8d59aa600cba9dd111d/sdks/java/transform-service/src/main/java/org/apache/beam/sdk/transformservice/ExpansionService.java#L159

Reply:
No. We don't actually delete anything, just copy the (hopefully) relevent bits to the bottom of the wall of text.

Reply:
Probably let's also add a unit test ?

Reply:
Added.